interfaces/GithubFactory.py

```diff
@@ -23,8 +26,11 @@ def get_issue(self, number: int):
         
         Returns:
         A tuple containing the Github issue object and the local Issue model object."""
-        
-        issue = self.repository.get_issue(number=number)
+        try:
+            issue = self.repository.get_issue(number=number)
+        except:
+            logging.exception(f"Could not find Issue with ID : {number}")
+            return 0, 0
```

Comment:
Dans quel cas on aurait un issue not found ?

Reply:
Je n'ai pas trouvé la cause également mais cela m'est déjà arrivé donc pour éviter les crashs, je préfère skip cette issue de manière générale